src/components/MainSidebar/ProgramDimensionsPanel/ProgramDimensionsPanel.js

```diff
@@ -131,46 +131,51 @@ const ProgramDimensionsPanel = ({ visible }) => {
 
     return (
         <div className={styles.container}>
-            <div className={cx(styles.section, styles.bordered)}>
-                <ProgramSelect
-                    programs={filteredPrograms}
-                    selectedProgram={selectedProgram}
-                    setSelectedProgramId={setSelectedProgramId}
-                    requiredStageSelection={requiredStageSelection}
-                />
-            </div>
-            <div
-                className={cx(styles.section, {
-                    [styles.bordered]: !!selectedProgramId,
-                })}
-            >
-                {isProgramSelectionComplete ? (
-                    <ProgramDimensionsFilter
-                        program={selectedProgram}
-                        searchTerm={searchTerm}
-                        setSearchTerm={setSearchTerm}
-                        dimensionType={dimensionType}
-                        setDimensionType={setDimensionType}
+            {visible && (
+                <div className={cx(styles.section, styles.bordered)}>
+                    <ProgramSelect
+                        programs={filteredPrograms}
+                        selectedProgram={selectedProgram}
+                        setSelectedProgramId={setSelectedProgramId}
+                        requiredStageSelection={requiredStageSelection}
                     />
-                ) : (
-                    <div className={styles.helptext}>
-                        {requiredStageSelection
-                            ? i18n.t(
-                                  'Choose a program and stage above to add program dimensions.'
-                              )
-                            : i18n.t(
-                                  'Choose a program above to add program dimensions.'
-                              )}
-                    </div>
-                )}
-            </div>
+                </div>
+            )}
+            {visible && (
```

Comment:
Could this div be part of the above condition?

Reply:
yep